guides/micronaut-mqtt/cli/groovy/src/test/groovy/example/micronaut/MicronautguideCommandSpec.groovy

```diff
@@ -0,0 +1,54 @@
+package example.micronaut
+
+import io.micronaut.configuration.picocli.PicocliRunner
+import io.micronaut.context.ApplicationContext
+import io.micronaut.context.annotation.Replaces
+import io.micronaut.context.env.Environment
+import jakarta.inject.Singleton
+import spock.lang.Specification
+
+import static java.nio.charset.StandardCharsets.UTF_8
+
+class MicronautguideCommandSpec extends Specification {
+
+    void testWithCommandLineOption() {
+        when:
+        OutputStream baos = new ByteArrayOutputStream()
+        System.setOut(new PrintStream(baos))
+
+        ApplicationContext ctx = ApplicationContext.run(
+                ['spec.name': 'MicronautguideCommandTest'],
```

Comment:
It needs to add `mqtt.enabled", false`. See https://github.com/micronaut-projects/micronaut-guides/pull/817/commits/1d422dc0aaa1a4df7e2cf07df5fa7ea7a249bd40